pkg/querier/store.go

```diff
@@ -53,13 +53,11 @@ func (q Querier) queryStore(ctx context.Context, req *logproto.QueryRequest) ([]
 
 func filterChunksByTime(from, through model.Time, chunks []chunk.Chunk) []chunk.Chunk {
 	filtered := make([]chunk.Chunk, 0, len(chunks))
```

Comment:
@gouthamve I think I've asked you a question about this during the PR, is it ok to remove ?

Reply:
Yep, this looks good!